cli/init/init.go

```diff
@@ -156,7 +159,14 @@ func generateTtEnv(configPath string, sourceCfg configData) error {
 	}
 	cfg.CliConfig.App.TarantoolctlLayout = sourceCfg.tarantoolctlLayout
 
-	if err := util.WriteYaml(configPath, cfg); err != nil {
+	template := string(ttYamlTemplate)
```

Comment:
If we want the template to be string, why don't we declare it as string on 135 line?

Reply:
Fixed.